README.md

```diff
@@ -496,12 +496,6 @@ Overrides the default PostgreSQL log directory.
 
 Default value: initdb's default path.
 
-##### `log_line_prefix`
```

Comment:
Yes.  It is still there, but now it is possible to use `server::config_entry` instead of setting it in here like all other configuration variables.  I meant to silently deprecate this usage by not documenting it anymore.